### PR TITLE
feat(devservices): Add relay configs

### PR DESCRIPTION
**Pull request by hubertdeng123**
This adds devservices configs to the relay repo. This allows devservices to handle dependencies of relay (redis, kafka) in order to aid in a smoother development environment. This is an initial configuration that will be subject to change as time goes on. 

#skip-changelog

### CODE REVIEW

File: devservices/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  modes:
+    default: [redis, kafka]
+
+services:
+  redis:
+    image: "redis:6.2.14-alpine"
+    healthcheck:
+      test: redis-cli ping
+  kafka:
+    image: "confluentinc/cp-kafka:7.6.1"
```
Comment (olksdr) [inline devservices/docker-compose.yml]: @hubertdeng123  how the redis and kafka versions will be kept in-sync with Sentry repo? 
Is it all manual open/merge PRs operation?

**Review by Dav1dde — APPROVED** [review]
Is the intention to be able to also just run this with `docker-compose` directly and `devservices`?

File: devservices/docker-compose.yml (hunk 1 above)
Comment (IanWoodard) [reply devservices/docker-compose.yml]: We are planning on making a shared definitions for kafka and redis to be used across Sentry services to ensure versions and other settings are kept in-sync

Comment (hubertdeng123) [reply devservices/docker-compose.yml]: Indeed, not available initially but it is something we are planning to add as we implement the logic for this.

**Comment by hubertdeng123** [comment]
> Is the intention to be able to also just run this with docker-compose directly and devservices?

With devservices, because due to implementation using docker compose could cause unwanted behavior. We'll change the config file name to avoid this confusion

**Pull request merged**